src/scripts/main.js

```diff
@@ -1 +1,68 @@
 'use strict';
+
+const createMessage = (text, type) => {
+  const newMessage = document.createElement('div');
+
+  newMessage.dataset.qa = 'notification';
+
+  newMessage.className = `message ${type}`;
+
+  newMessage.textContent = text;
+
+  document.body.append(newMessage);
+};
+
+const firstPromise = new Promise((resolve, reject) => {
+  let click = false;
+
+  document.addEventListener('click', e => {
+    click = true;
+
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    if (click === false) {
+      reject(new Error('First promise was rejected'));
+    }
+  }, 3000);
+});
+
+const secondPromise = new Promise((resolve) => {
+  document.addEventListener('contextmenu', e => {
+    resolve('Second promise was resolved');
+  });
+
+  document.addEventListener('click', e => {
+    resolve('Second promise was resolved');
+  });
```

Comment:
you didn't use this event
```suggestion
  document.addEventListener('contextmenu', () => {
    resolve('Second promise was resolved');
  });

  document.addEventListener('click', () => {
    resolve('Second promise was resolved');
  });
```